packages/gasket-plugin-nextjs/lib/build.js

```diff
@@ -0,0 +1,42 @@
+const path = require('path');
+const { promisify } = require('util');
+const exec = promisify(require('child_process').exec);
+const builder = require('next/dist/build').default;
+
+const { createConfig } = require('./config');
+
+/**
+ * Checks if Next.js Telemetry is enabled. If so, it disables it.
+ *
+ * https://nextjs.org/telemetry#how-do-i-opt-out
+ *
+ * @param {object} logger gasket logger
+ */
+async function checkTelemetryStatus(logger) {
+  const statusRegex = /(?<=Status: ).*./;
+  const { stdout = '' } = await exec('npx next telemetry status');
+  const telemetryStatus = stdout.match(statusRegex)[0] || '';
+
+  if (telemetryStatus === 'Enabled') {
+    const { stdout: disableCmdOutput = '' } = await exec('npx next telemetry disable');
+    logger.info(`Next.js Telemetry: ${disableCmdOutput.match(statusRegex)[0]}`);
+  }
```

Comment:
is disabling slow vs the status check? should we just always run disable?

Can we make use fo the env variable instead and just set `NEXT_TELEMETRY_DISABLED=1`

Reply:
@msluther I haven't seen or read about any sort of performance impact if we were to just run disable, so that may be a better/simpler direction here.

I think the env variable direction may actually be a better option.